worker/upgradesteps/worker.go

```diff
@@ -27,26 +27,29 @@ import (
 	"github.com/juju/juju/wrench"
 )
 
-var logger = loggo.GetLogger("juju.worker.upgradesteps")
+var logger = loggo.GetLogger("juju.worker.upgradeSteps")
```

Comment:
nit: this should be lowercase to reflect the package name.

Reply:
Good catch. Fixed.